model/utils/mlflow.py

```diff
@@ -17,29 +15,27 @@ def close(self):
         mlflow.end_run()
 
     def login_to_mlflow(
-        self, uri: str = "databricks", experiment_name: str = "/Users/michal.zareba@softwaremill.pl/housepricesml"
+        self, uri: str = "http://48.209.80.111:5000", experiment_name: str = "housepricesml"
```

Comment:
I would suggest having this in constructor of an object. And second thing, I would recommend fetching this data from Environment variables or from config file, e.g. `uri = os.environ.get("MLFLOW_TRACKING_URI", "http://48.209.80.111:5000")`